server.js

```diff
@@ -155,7 +155,9 @@ if (AppConstants.FXA_ENABLED) {
 app.use("/scan", ScanRoutes);
 app.use("/ses", SesRoutes);
 app.use("/user", UserRoutes);
-app.use("/email-l10n", EmailL10nRoutes);
+if (app.get("env") === "dev" || AppConstants.NODE_ENV === "heroku") {
```

Comment:
Hmm ... `app.get("env")` _should_ be the same as `AppConstants.NODE_ENV`, but can we keep this consistent? So:
```
if (["dev", "heroku"].includes(AppConstants.NODE_ENV)) {
```

Reply:
Works! I think it will mean that anyone poking around on email-l10n routes locally will either need a self-signed cert for their localhost or will just need to disable the https redirect.